pos_frequency_pipeline.py

```diff
@@ -2,7 +2,39 @@
 Implementation of POSFrequencyPipeline for score ten only.
 """
 
+from collections import defaultdict
+import json
+from pathlib import Path
+import re
+
+from constants import ASSETS_PATH
+from visualizer import visualize
+
 
 class POSFrequencyPipeline:
-    def __init__(self, assets):
-        pass
+    def __init__(self, corpus_manager):
+        self.corpus_manager = corpus_manager
+
+    def run(self):
+        """
+        Runs pipeline visualize scenario
+        """
+        articles = self.corpus_manager.get_articles()
+        for article in articles.values():
+            processed_text_path = Path(ASSETS_PATH) / f'{article.article_id}_processed.txt'
+            with open(processed_text_path, encoding='utf-8') as file:
+                processed_text = file.read()
+
+            pos_tags = defaultdict(int)
+            tags = re.findall('<(.+?)>', processed_text)
```

Comment:
we could directly extract POS by the regex

Reply:
fixied